docs/java-api/client.asciidoc

```diff
@@ -167,7 +53,13 @@ Client client = TransportClient.builder().settings(settings).build();
 //Add transport addresses and do something with the client...
 --------------------------------------------------
 
-Or using `elasticsearch.yml` file as shown in <<node-client>>
+Or you can define `cluster.name` in the `/src/main/resources/elasticsearch.yml` file in your project.
+As long as `elasticsearch.yml` is present in the classpath, it will be used when you start your client.
```

Comment:
So this will be removed when I'll rebase on master. See https://github.com/elastic/elasticsearch/commit/1769e9b78a5f8c17aa5ca5d8f00916a6ac96cfb8